src/components/Individual_Article/List_of_Comments/List_of_Comments.jsx

```diff
@@ -0,0 +1,37 @@
+import { useState, useEffect } from "react"
+import { getArticleComments } from "../../../../api"
+
+const CommentsList = ({articleId}) => {
+    const [articleComments, setArticleComments] = useState([])
+    const [isLoading, setLoading] = useState(true)
+    
+    if(articleId){
+        useEffect (() => {
+            getArticleComments(articleId).then((fetchedComments) => {
+                setArticleComments(fetchedComments)
+                setLoading(false)
+            })
+        }, [])
+    }
+    if (isLoading) return <p>Loading...</p>
+
+    return (
+        <section id="commentsSection">
+            <h3 id='commentsHeading'>Comments</h3>
+            <ul id="commentsList">
+                {articleComments.map((comment) => {
+                    return <li key={comment.comment_id}>
+                            <section className='commentGrid'>
+                            <p className='commentBody' key={comment.body}>{comment.body}</p> 
+                            <p className='commentVotes' key={comment.votes}>{comment.votes} likes</p> 
+                            <p className='commentAuthor' key={comment.author}>{comment.author}</p> 
+                            <p className='commentCreated' key={comment.created_at}>{comment.created_at}</p> 
```

Comment:
It would be cool if the date was a little bit more readable 📆 
You could either do this using vanilla javascript or you could use a library like: https://day.js.org/